src/ansys/mapdl/core/launcher.py

```diff
@@ -1410,6 +1437,9 @@ def launch_mapdl(
     if ip is None:
         ip = os.environ.get("PYMAPDL_IP", LOCALHOST)
     else:  # pragma: no cover
+        LOG.debug(
+            "Because IP is not None, we are going to attempt to connect to a remote session. ('START_INSTANCE' is set to False)"
```

Comment:
```suggestion
            "Because ``PYMAPDL_IP is not ``None``, an attempt is made to connect to a remote session. ('START_INSTANCE' is set to ``False``.)"
```
This wording change likely affects a test?

Reply:
I think it does not. :)